specification/hdinsight/resource-manager/Microsoft.HDInsight/preview/2015-03-01-preview/applications.json

```diff
@@ -249,15 +301,20 @@
         "publicPort": {
           "type": "integer",
           "format": "int32",
-          "description": "The public port to connect to."
+          "description": "The public port to connect to.",
+          "readOnly": true
+        },
+        "privateIPAddress": {
```

Comment:
Are you adding this new property now? Or just fixing the S360 violation?

Reply:
> Are you adding this new property now? Or just fixing the S360 violation?

Hi @RamyasreeChakka  Thanks for your comment.
The response from backend includes this property "privateIPAddress" but it hasn't been defined in our swagger spec. Adding this new property is for fixing S360 Issues.